README.rst

```diff
@@ -64,6 +66,77 @@ To check if everything worked fine, you can do::
 
 and it should not give any error messages.
 
+.. _Parallel backends:
+
+Parallel backends
+=================
+
+Two options are available for making use of multiple CPU cores. The first runs multiple trials in parallel with joblib. Alternatively, you can run each trial across multiple cores to reduce the runtime.
+
+Joblib
+------
+
+This is the default backend and will execute multiple trials at the same time, with each trial running on a separate core in "embarrassingly parallel" execution. Note that with only 1 trial, there will be no parallelism.
+
+**Dependencies**::
+
+	$ pip install joblib
+
+**Usage**::
+
+	dpls = simulate_dipole(net, n_trials=2)
+
+MPI
+------
+
+This backend will use MPI (Message Passing Interface) on the system to split neurons across CPU cores (processors) and reduce the simulation time as more cores are used.
+
+**Linux Dependencies**::
+
+	$ sudo apt-get install libopenmpi-dev openmpi-bin
+	$ pip install mpi4py psutil
+
+**MacOS Dependencies**::
+
+	$ conda install yes openmpi mpi4py
+	$ pip install psutil
+
+**MacOS Environment**::
+
+	$ export LD_LIBRARY_PATH=${CONDA_PREFIX}/lib
+
+Alternatively, run the commands below will avoid needing to run the export command every time a new shell is opened::
+
+	cd ${CONDA_PREFIX}
+	mkdir -p etc/conda/activate.d etc/conda/deactivate.d
```

Comment:
This should be:

```suggestion
	mkdir -p /etc/conda/activate.d etc/conda/deactivate.d
```

I think?

Reply:
The path is supposed to be `${CONDA_PREFIX}/etc/conda/activate.d`

Reply:
indeed :-)

Reply:
which is accomplished by the current code